pkg/tfbridge/schema_test.go

```diff
@@ -396,10 +396,31 @@ func TestTerraformOutputsWithSecretsSupported(t *testing.T) {
 		},
 		{
 			name: "object_property_value",
+
+			// NOTE: This input does not match the schema's type. The tfValue *should* be wrapped
+			// in a []any{}. [MakeTerraformOutputs] handles this case, but it shouldn't need to
+			// and tf should never return it.
 			tfValue: map[string]interface{}{
 				"property_a": "a",
 				"property_b": true,
 			},
+			tfType: &schema.Schema{
```

Comment:
Hmm, strange test case it didn't have tfType before so it was trying to parse untyped?